addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -877,9 +921,13 @@ registry.backgroundVideo = publicWidget.Widget.extend(MobileYoutubeAutoplayMixin
      * @private
      */
     _appendBgVideo: function () {
+        const allowedCookies = !this.el.dataset.needCookiesApproval;
+
         var $oldContainer = this.$bgVideoContainer || this.$('> .o_bg_video_container');
         this.$bgVideoContainer = $(renderToElement('website.background.video', {
-            videoSrc: this.videoSrc,
+            videoSrc: allowedCookies
+                      ? this.videoSrc
+                      : this._getPrivacyFriendlyVideoLink(this.videoSrc),
```

Comment:
Removing `_getPrivacyFriendlyVideoLink` would remove this diff (and the extend of `videoNoCookieMixin` above)

Reply:
Not entirely: I still need to remove the link if the cookies are not allowed (but no more mixin indeed).